src/pybind/mgr/mgr_util.py

```diff
@@ -335,6 +338,116 @@ def get_all_filesystems(self) -> List[str]:
         return fs_list
 
 
+class CephFsEarmarkResolver:
```

Comment:
Naming nitpick: 'CephFSEarmarkResolver' not 'CephFsEarmarkResolver'. File system is typically two words in general practice.